### PR TITLE
CORE-8981 Add `OLS Ontology Term` to the `value_types` table.

### PR DESCRIPTION
**Pull request by psarando**
This PR will add the `OLS Ontology Term` to the `value_types` table.
It also imports the `data/*value_types.sql` script from cyverse-de/de-db@2a757d13ea18f1feb039f70ac8910ba4ed790485.

### CODE REVIEW

File: src/main/conversions/c2_15_0_2017081601.clj [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    (:use [korma.core :exclude [update]]
+          [kameleon.sql-reader :only [exec-sql-statement]]
+          [kameleon.uuids]))
+
+(def ^:private version
+  "The destination database version"
+  "2.15.0:20170816.01")
+
+(defn- add-ols-ontology-term-value-type
+  "Updates the value_types view."
```
Comment (ianmcorvidae) [inline src/main/conversions/c2_15_0_2017081601.clj]: This is hyper-nitpicky but `value_types` isn't a view!

**Review by ianmcorvidae — APPROVED** [review]
Other than one comment which is entirely without functional relevance, looks good to me!

File: src/main/conversions/c2_15_0_2017081601.clj (hunk 1 above)
Comment (psarando) [reply src/main/conversions/c2_15_0_2017081601.clj]: Nope, good catch 😆

**Comment by psarando** [comment]
Thanks for the review 👍

**Pull request merged**